docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -125,6 +125,21 @@ components. This may take several minutes.
 - You can't run `kubeadm init` twice without tearing down the cluster in between
 ([unless you're upgrading from v1.6 to v1.7](/docs/tasks/administer-cluster/kubeadm-upgrade-1-7/)),
 see [Tear Down](#tear-down).
+- Optionally, to specify IPV6 for liveness probe for control-plan components and etcd server, you can specify extraArgs in kubeadm.conf file as mentioned in example below. ${APISERVER_ADVERTISE_ADDRESS} is your IPV6 enclosed in square brackets.
```

Comment:
Substitute control-plan with controlplane or control plane
Substitute IPV6 with IPv6

Instead of adding line 128. I would add to line 119. Line 119 is already referring to using custom args to customize the controlplane.

Then update /docs/admin/kubeadm#custom-args with your example kubeadm.conf with a sentence that states this is an example of setting an IPv6 address for controller-manager, scheduler and etcd. Replace ${APISERVER_ADVERTISE_ADDRESS} with an example IPv6 apiserver address.